_shared/project/.github/workflows/ci.yml

```diff
@@ -40,10 +40,8 @@ on:
 {% endif %}
   workflow_dispatch:
   workflow_call:
-  {% if cookiecutter._directory == 'pypackage' %}
   schedule:
   - cron: '0 1 * * *'
-  {% endif %}
```

Comment:
Let's run `ci.yml` on `main` nightly for all our projects, not just packages. It's less necessary for apps since they have pinned dependencies and receive Dependabot PRs but I think the nightly run is still useful to provide a history of CI runs on `main` so we can see if it's passing or not and whether it broke.